easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.4.4-iomkl-2016.09-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,20 @@
+name = 'netCDF-Fortran'
+version = '4.4.4'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented
+ scientific data."""
+
+toolchain = {'name': 'iomkl', 'version': '2016.09-GCC-4.9.3-2.25'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'ftp://ftp.unidata.ucar.edu/pub/netcdf/',
```

Comment:
please use `https://github.com/Unidata/netcdf-c/archive/` and tweak sources to `v%(version)s.tar.gz`